modular_skyrat/modules/gun_quirk/quirk.dm

```diff
@@ -0,0 +1,55 @@
+///Mail goodie list.
+#define GUN_MAIL_LIST list(	/obj/item/gun_maintenance_supplies,\
+							/obj/item/bedsheet/patriot,\
+							/obj/item/clothing/mask/gas/hecu2,\
+							/obj/item/storage/backpack/industrial/cin_surplus/random_color,\
+							/obj/item/storage/belt/military/cin_surplus/random_color,\
+							/obj/item/toy/gun,\
+							/obj/item/toy/ammo/gun,\
+							)
+
+///Gun you get on spawn.
+#define STARTER_GUN_LIST list(/obj/item/gun/ballistic/automatic/pistol/g17,\
+							/obj/item/gun/ballistic/automatic/pistol/mk58,\
+							/obj/item/gun/ballistic/automatic/pistol/pdh/peacekeeper,\
+							/obj/item/gun/ballistic/automatic/dozer,\
+							/obj/item/gun/ballistic/automatic/pistol/cfa_snub,\
+							)
```

Comment:
Not really sold on giving people the equivalent of a chest rig for free through mail loot just from an already extremely strong quirk.

Also, please, try to format your lists properly, it's not the first time that I've had to review your code and that there's lists that aren't formatted properly.

The dozer and the CFA also feel significantly stronger than the other three options, so I'm not convinced they're a good pick for the balance of this.
```suggestion
#define GUN_MAIL_LIST list(
	/obj/item/gun_maintenance_supplies, \
	/obj/item/bedsheet/patriot, \
	/obj/item/clothing/mask/gas/hecu2, \
	/obj/item/storage/backpack/industrial/cin_surplus/random_color, \
	/obj/item/toy/gun, \
	/obj/item/toy/ammo/gun, \
)

///Gun you get on spawn.
#define STARTER_GUN_LIST list(
	/obj/item/gun/ballistic/automatic/pistol/g17, \
	/obj/item/gun/ballistic/automatic/pistol/mk58, \
	/obj/item/gun/ballistic/automatic/pistol/pdh/peacekeeper, \
)
```